src/app/_schemas/SeoMetadataSchema.ts

```diff
@@ -6,27 +6,31 @@ const { seo_metadata_description_max_characters } = configJson
 
 const TwitterCardType = z.enum(['summary', 'summary_large_image', 'player'])
 
+const OpenGraphMetadataSchema = z
+  .object({
+    title: z.string().optional(),
+    description: z.string().optional(),
+    image: z.string().optional(),
+  })
+  .strict()
+  .optional()
+
+const TwitterMetadataSchema = z
+  .object({
+    card: TwitterCardType.optional(),
+    site: z.string().optional(),
+    creator: z.string().optional(),
+  })
+  .strict()
+  .optional()
+
 export const SeoMetadataSchema = z
   .object({
     title: z.string(),
```

Comment:
Don't we need (another) schema where the title is optional? Right now, if you remove `title: Letter from the Guest Editor` from the below, Zod will yell at us.

```
seo:
  title: Letter from the Guest Editor
  description: "Dear Readers, Welcome to the inaugural edition of The Filecoin
    Ecosystem Digest: All Systems Go - a journey into the transformative power
    of the decentralized web and its far-reaching implications across
    various..."
```